docs/docs/events/overview.md

```diff
@@ -18,12 +18,17 @@ Tracing `execve` events with [policies]:
 
 ```
 cat <<EOF >sample_policy.yaml
-name: sample policy
-description: traces execve events
-scope:
-  - global
-rules:
-  - event: execve
+apiVersion: aquasecurity.github.io/v1beta1
+kind: TraceePolicy
+metadata:
+	name: sample policy
```

Comment:
Not directly related to this PR but we should consider enforcing policy names without spaces (which may cause parsing issues when listed in the event output), and maybe also limit its size

Reply:
I created an issue to keep track of it https://github.com/aquasecurity/tracee/issues/3333